common/src/test/scala/uk/ac/wellcome/test/utils/SQSLocal.scala

```diff
@@ -16,12 +24,28 @@ trait SQSLocal extends Suite with BeforeAndAfterEach {
       new EndpointConfiguration(s"http://localhost:9324", "localhost"))
     .build()
 
-  val idMinterQueue = "es_id_minter_queue"
-  val idMinterQueueUrl = sqsClient.createQueue(idMinterQueue).getQueueUrl
+  def queueName: String
+
+  // Use eventually to allow some time for the local SQS to start up. Seems to be needed on CI
```

Comment:
> Seems to be needed on CI

Can you be more specific? What do we see if we don't use `eventually`? Does it hang? Crash? Fail prematurely?